docs/reference/deploy/master-config-reference.rst

```diff
@@ -232,6 +232,20 @@ otherwise active (as defined by the ``notebook_idle_type`` option in the :ref:`t
 
 The resource manager used to acquire resources. Defaults to ``agent``.
 
+If additional resource managers are defined (only for Kubernetes), the resource manager defined
+here, under the primary ``resource_manager`` key, is considered the default resource manager.
+
+``name``
+========
+
+Optional. The resource manager name to use. Defaults to "default" if no name is specified.
+
+``metadata``
+============
+
+Optional. A place to store additional information on a resource manager, like the zone, region,
+location etc.
```

Comment:
``metadata``
============

Optional. Stores additional information about the resource manager, such as the zone, region, or location.